.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -109,6 +110,36 @@ fun BaseGradleBuildType.gradleRunnerStep(model: CIBuildModel, gradleTasks: Strin
     }
 }
 
+private
+fun BuildType.attachFileLeakDetector() {
+    steps {
+        script {
+            name = "ATTACH_FILE_LEAK_DETECTOR"
+            executionMode = BuildStep.ExecutionMode.ALWAYS
+            scriptContent = """
+            del gradle.properties
+            del buildSrc\gradle.properties
+            rename gradle.windows.properties gradle.properties
```

Comment:
:o: Instead of having the second file - `gradle.windows.properties` - could the script here do a text replace in the existing file adding `javaagent:C:\\tcagent1\\...` to the `org.gradle.jvmargs`?

With the two files, I think any update of `gradle.properties` might easily forget to update `gradle.windows.properties`. And we might not notice that for a long time until we run into some unexpected behavior.

Reply:
Sure, let me try to find a `sed`-equivalent on windows.